packages/dev-utils/src/chain-setup.ts

```diff
@@ -1,5 +1,4 @@
-import { newGovernance } from '@celo/abis/web3/Governance'
-import { newValidators } from '@celo/abis/web3/Validators'
+import { governanceABI, validatorsABI } from '@celo/abis'
```

Comment:
why this change?

Reply:
I just can't seem to import it properly in dev-utils. replied on slack